meritocracy/backend/rest_test.sh

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env bash
```

Comment:
There are so many good test runners, so i suggest we use one, instead of homebuilt shell script.
Tests should be of form expected value, receieved value - should not print values unless they fail
you can use either dbunit or httpunit (to test via postgrest api)

Reply:
"Prototype before polish". I have no problem replacing this with something fancier going forward.